### PR TITLE
Fix: fixed the aligmnent of learning outcome (circles)

### PR DESCRIPTION
**Pull request by TejaswiniMannuru24**
This is a fix to the card: https://trello.com/c/tSBYCpdn

### CODE REVIEW

**Comment by macite** [comment]
I think we want to redesign it more fully... Something smaller would be better. No need to be 5 separate filled/empty things...

**Comment by macite** [comment]
Next time it would be good to provide a screenshot showing the change here along with the text details.

**Comment by jakerenzella** [comment]
@macite I asked for this just so I could check it, we were discussing making it smaller

**Comment by macite** [comment]
@jakerenzella can you review and merge when happy

**Comment by macite** [comment]
1: So we would remove the overdue badge from the `do not resubmit`, `time exceeded`, `discuss`, and `demonstrate` - sounds ok.
2: Agreed - we can hide this if submitted
3: :+1:
4: Ok

Looks good - but this is now in with the changes to the alignment weight :(

File: src/app/tasks/task-ilo-alignment/task-ilo-alignment-rater/task-ilo-alignment-rater.scss [hunk 1]
```diff
@@ -28,7 +28,9 @@
     }
     width: 100%;
     text-align: center;
-    display: inline-block;
+    /*display: inline-block;*/
+    display: flex; /* Teju */
```
[the remark this answers is not in the document]
Comment (jakerenzella) [reply src/app/tasks/task-ilo-alignment/task-ilo-alignment-rater/task-ilo-alignment-rater.scss]: This is a good fix for now -- but we still need to redesign the entire card -- let's keep that as a separate PR though. Good work.

**Comment by jakerenzella** [comment]
> Instead, a message to indicate that the task is pending with tutor to mark and students need not take any action will avoid the confusion.
> 
> after

I don't think we should add a new card here... Just update the text from the first description and remove the (upload new files) button... we have too many cards I think.

File: src/app/common/services/task-service.coffee [hunk 2]
```diff
@@ -58,15 +58,15 @@ angular.module("doubtfire.common.services.tasks", [])
   # Which states can a task be considered to be overdue
   taskService.overdueStates = [
```
[the remark this answers is not in the document]
Comment (macite) [reply src/app/common/services/task-service.coffee]: Decided to keep in PR for simplicity

**Comment by macite** [comment]
Any progress on this @jakerenzella?

**Pull request merged**